src/app/shared/services/iam.service.ts

```diff
@@ -95,22 +84,7 @@ export class IamService {
       this.signerService = signerService;
       this.messagingService = messagingService;
       if (initCacheServer) {
```

Comment:
Instead of setting initCacheServer = true, use `envService.checkStakingVerification` and toggling cache server should work fine.